python/tvm/relay/frontend/tflite.py

```diff
@@ -1212,6 +1214,7 @@ def convert_conv(self, op, conv_type):
                                      input_zero_point=new_input_zero_point,
                                      output_scale=output_tensor.qnn_params['scale'],
                                      output_zero_point=output_tensor.qnn_params['zero_point'],
+                                     rounding="TFLITE",
```

Comment:
As the comment said, we could make the `rounding` is an argument of `from_tflite`, the default value is `TFLITE`. i.e. 
```python
def from_tflite(model, shape_dict, dtype_dict, rounding="TFLITE"):
```
Of course, you should pass the `rounding` to `OperatorConverter` as one of its member value so that other `convert_xxx` functions could get the value

Reply:
thx, resolved